aosp_xenvm_trout_arm64.mk

```diff
@@ -40,6 +40,7 @@ endif
 
 LOCAL_OEMLOCK_PRODUCT_PACKAGE := android.hardware.oemlock-service.example
 
+LOCAL_VHAL_PRODUCT_PACKAGE = android.hardware.automotive.vehicle@2.0-virtualization-service
```

Comment:
Hi @arminn, please comment that this line should be placed before the aosp_trout_arm64.mk included in Android 14.